frontend/app/src/components/ui/label.tsx

```diff
@@ -8,12 +8,27 @@ export interface LabelProps
     VariantProps<typeof labelVariants> {}
 
 const labelVariants = cva(
-  "text-sm font-medium text-gray-900 cursor-pointer peer-disabled:cursor-not-allowed peer-disabled:opacity-70"
+  "cursor-pointer peer-disabled:cursor-not-allowed peer-disabled:opacity-70",
+  {
+    variants: {
+      variant: {
+        noramal: "text-sm font-medium text-gray-900",
```

Comment:
typo noramal -> normal

I would even suggest to rename it to `default`, to be consistent with others components